.github/workflows/android-app-build.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       matrix:
         android-api: [26, 34]
         android-abi: [x86_64]
-        os: [macos-12, ubuntu-22.04]
+        os: [macos-13, ubuntu-22.04]
```

Comment:
Did you try `macos-14`?

Reply:
macOS-14 may introduce a new issue: 'Timeout waiting for emulator to boot.' Currently, there is no known solution to address this issue.

https://github.com/openthread/ot-commissioner/actions/runs/12174524147?pr=308

Reply:
Thanks for checking. I'm fine with deferring change to `macos-14` to later.